pkgs/development/python-modules/patool/default.nix

```diff
@@ -6,6 +6,14 @@
 # it will still use unrar if present in the path
 
 let
+  # FIXME: backport a patch in `file` that seemingly only affects this package
+  # Revert when fix to main package makes it through staging.
+  # Original commit: https://github.com/file/file/commit/1dd21dd360472d7b830825df8e40a06cdc1cbbcf.patch
+  # Hacked up to apply to stable release.
+  file' = file.overrideAttrs(old: {
+    patches = (old.patches or []) ++ [ ./file-regression-fix.patch ];
```

Comment:
We could `fetchurl` this patch from Gentoo: https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-apps/file/files/file-5.44-decompress-empty.patch?h=dfc57da515a2aaf085bea68267cc727f1bfaa691